mobile-app/app/screens/AppNavigator/screens/Balances/screens/GetDFIScreen.tsx

```diff
@@ -0,0 +1,431 @@
+import * as Clipboard from 'expo-clipboard'
+import { useCallback, useEffect, useState } from 'react'
+import { ImageSourcePropType, Share, TouchableOpacity, View, Image } from 'react-native'
+import QRCode from 'react-native-qrcode-svg'
+import { ThemedIcon, ThemedScrollView, ThemedText, ThemedTouchableOpacity, ThemedView } from '@components/themed'
+import { useToast } from 'react-native-toast-notifications'
+import { useThemeContext } from '@shared-contexts/ThemeProvider'
+import { useWalletContext } from '@shared-contexts/WalletContext'
+import { tailwind } from '@tailwind'
+import { translate } from '@translations'
+import { NativeLoggingProps, useLogger } from '@shared-contexts/NativeLoggingProvider'
+import { debounce } from 'lodash'
+import { openURL } from '@api/linking'
+import { IconTooltip } from '@components/tooltip/IconTooltip'
+import NumberFormat from 'react-number-format'
+import BigNumber from 'bignumber.js'
+import { useSelector } from 'react-redux'
+import { RootState } from '@store'
+import { useWhaleApiClient } from '@shared-contexts/WhaleContext'
+import Kucoin from '@assets/images/exchanges/Kucoin.png'
+import Bittrex from '@assets/images/exchanges/Bittrex.png'
+import Bitrue from '@assets/images/exchanges/Bitrue.png'
+import Latoken from '@assets/images/exchanges/Latoken.png'
+import DFX from '@assets/images/exchanges/DFX.png'
+import Transak from '@assets/images/exchanges/Transak.png'
+import Hotbit from '@assets/images/exchanges/Hotbit.png'
+import Hoo from '@assets/images/exchanges/Hoo.png'
+import EasyCrypto from '@assets/images/exchanges/EasyCrypto.png'
+import CakeDeFi from '@assets/images/exchanges/CakeDeFi.png'
+
+interface ExchangeProps {
+  image: ImageSourcePropType
+  name: string
+  url: string
+}
+
+const exchanges: ExchangeProps[] = [
+  {
+    name: 'Kucoin',
+    image: Kucoin,
+    url: 'https://www.kucoin.com/trade/DFI-BTC'
```

Comment:
When one or more of these urls become invalid in the future, it would take us a day or two to update/remove it because of app reviews. Is this ok?  CC: @fullstackninja864 @thedoublejay

Reply:
Let's keep it for now